modules/programs/bash.nix

```diff
@@ -157,7 +157,7 @@ in
       historyControlStr =
         concatStringsSep "\n" (mapAttrsToList (n: v: "${n}=${v}") (
           {
-            HISTFILE = "\"${cfg.historyFile}\"";
```

Comment:
Why move the quotes to default value in `mkOption`? Seems to me like it would work to keep them here. It would also keep working for anybody who for some reason has a space in their history file path.

Reply:
Whoops, that was a mistake. Should have looked at the old implementation before reverting my changes. Will move them back.